copilot-instructions.md

```diff
@@ -0,0 +1,408 @@
+# GitHub Copilot Instructions for Sitecore Content SDK
+
+## Project Overview
+
+This is the **Sitecore Content SDK** - a comprehensive TypeScript/JavaScript SDK for building modern web applications with Sitecore XM Cloud. The project provides core functionality, React components, Next.js integration, and CLI tools for scaffolding applications.
+
+### Tech Stack
+- **Language**: TypeScript (Node LTS)
+- **Runtime**: Node LTS or higher
```

Comment:
We only rely on LTS version and don't really use "latest", probably, let's update it
```suggestion
- **Runtime**: Node LTS
```